easybuild/easyconfigs/a/ant/ant-fix-doclint-error.patch

```diff
@@ -0,0 +1,11 @@
+diff -aur a/build.xml b/build.xml
```

Comment:
@Flamefire Please include a brief patch description, with references to relevant upstream bug issues or PRs (or a patch author)

Reply:
"Author" is me, not sure if it is worth mentioning for 1 line (re)added ;)
Where/how would I add the description?

Reply:
Just on top, anything above the 1st `diff` line is comments.

Reply:
Ok, done